File: mario.c
#include <stdio.h>
#include <cs50.h>

int i, a, b, c;

int main(void)
{
do  {
    printf("Number of rows = ");
    i = get_int();
    }
while (i < 0 || i > 23);

for (a = 0; a < i; a++) //counts i times
{
    for (b = 0; b < i - a - 1; b++)
    {
    printf(" ");
    }
    for (c = 0; c < a; c++)
    {
    printf("#");
    }

    printf("##" "\n");
}
}

File: mario.py
import cs50

def main():
    while True:
        print("i = ", end="")
        i = cs50.get_int()
        if i > 0 or i < 23:
            break

    for a in range(i):
        for b in range(i - a - 1):
            print(" ", end="")
        for c in range(a):
            print("#", end="")
        print("##")

if __name__ == "__main__":
    main()